github_ops/client.py

```
@@ -88,259 +90,228 @@ def _check_rate_limit(self):
-
-                if "X-RateLimit-Remaining" in response.headers:
-                    self.rate_limit_remaining = int(
-                        response.headers["X-RateLimit-Remaining"]
-                    )
-                    self.rate_limit_reset_time = int(
-                        response.headers["X-RateLimit-Reset"]
-                    )
-
-                return response.json()
-
+                return resp
```

Ruff (0.15.5)

[warning] Consider moving this statement to an `else` block

(TRY300)